packages/ovm/src/app/abi.ts

```diff
@@ -0,0 +1,17 @@
+/* External Imports */
+import { utils, ContractFactory } from 'ethers'
+
+/* Internal Imports */
+import * as ExecutionManager from '../../build/contracts/ExecutionManager.json'
+
+const EMContract = new ContractFactory(
+  ExecutionManager.abi,
+  ExecutionManager.bytecode
+)
+const EMEvents = EMContract.interface.events
+const topics = []
+for (const eventKey of Object.keys(EMEvents)) {
+  topics.push(EMEvents[eventKey].topic)
+}
+
+export const ALL_EXECUTION_MANAGER_EVENT_TOPICS = topics
```

Comment:
could this go in utils.ts? It has and exports abi stuff. If nothing else, maybe reference it instead of importing the contract?

Reply:
Happy to move to utils, but what do you mean by "reference instead of import"?

Reply:
Just to export from [here](https://github.com/ethereum-optimism/optimism-monorepo/blob/synthetix-uat/packages/ovm/src/app/utils.ts#L33) and reference. Not a big priority though.